### DISCUSSION TITLE
picking the wrong beta suffix number

### DISCUSSION THREAD

**Discussion by hughrawlinson** [semantic-release/semantic-release#2217] [Q&A] [answered]
Hi!

I hope this is the right place to register this issue, if not, sorry!!

I'm having an issue where I've previously released and tagged `v6.0.0-beta.1` of my repo, and I've configured semantic release to release beta builds from my `v6`branch. The commit analyzer incorrectly registers the next version as `v6.0.0-beta.1` and then fails to release. My understanding is that it calculates the next version number based on the git tags, but it seems to not be taking that tag into account.

I imagine it's a misconfiguration on my end, sorry if that's the case. The failure log is [here](https://github.com/meyda/meyda/runs/4059587394?check_suite_focus=true), and the existing tagged commit is [here](https://github.com/meyda/meyda/releases/tag/v6.0.0-beta.1). If you have any recommendation on what to check next, I would greatly appreciate that!

Thanks for all the great work!

**Comment by travi** [comment, marked as answer]
git notes are used in addition to git tags. have you tried [these troubleshooting steps](https://semantic-release.gitbook.io/semantic-release/support/troubleshooting#release-not-found-release-branch-after-git-push-force)?

**Reply by hughrawlinson** [reply]
Thanks! That seems like the thing I needed that I didn't know I needed. It definitely did something - now it's not failing in the same way.

Now it seems to be attempting to actually publish over the existing version, even though I've created the tag, the note, and pushed both. I guess it succeeds in creating a tag, weirdly, because [the tag definitely already exists](https://github.com/meyda/meyda/releases/tag/v6.0.0-beta.1). Or it sees that the tag exists and then attempts to publish that tag?

**Reply by hughrawlinson** [reply]
I followed the instructions again today, it still tries to create a new tag despite one already existing. Any clues as to what might be happening?

**Reply by hughrawlinson** [reply]
Sorry, user error, I had channel as 'beta', not as 'v6'. Figured it out after 5 hours reading through the semantic release code lol.
